src/renderer/components/uielements/button/ViewTxButton.style.tsx

```diff
@@ -0,0 +1,18 @@
+import React from 'react'
+
+import styled from 'styled-components'
+import { palette } from 'styled-theme'
+
+import { Button as UIButton } from '.'
+import { ExternalLinkIcon as UIExternalLinkIcon } from '../common/Common.style'
+
+const ExternalLinkIcon = styled(UIExternalLinkIcon)`
+  svg {
+    color: ${palette('primary', 0)};
+  }
+`
+
+export const ViewTxButton = styled(UIButton).attrs({
+  typevalue: 'transparent',
+  icon: <ExternalLinkIcon />
+})``
```

Comment:
maybe we should handle disabled state for styles too ?

Reply:
`UIButton` has already some styles for `disabled` state or do you want to add different styles, here?